SDK/MobileCenter/WatsonRegistrationUtility/WatsonRegistrationManager.h

```diff
@@ -0,0 +1,11 @@
+﻿#pragma once
+
+namespace WatsonRegistrationUtility
+{
+	public ref class WatsonRegistrationManager sealed
```

Comment:
Why is public keyword on same indentation level as opening curly brace? Is that a c++ convention?